src/cattr/converters.py

```diff
@@ -240,7 +245,7 @@ def _structure_default(self, obj, cl):
         Bare optionals end here too (optionals with arguments are unions.) We
         treat bare optionals as Any.
         """
-        if cl is Any or cl is Optional:
+        if cl is Any or cl is Optional or cl is None:
```

Comment:
This feels suspect, why is the type none from hypothesis?

Reply:
Interesting, you're saying Hypothesis generates Nones for the type annotations? Not sure why off the top of my head. Which tests get it?